frontendservice/src/views/About.vue

```diff
@@ -94,6 +97,7 @@ export default {
       isLoggedIn: false,
       currentUser: false,
       isVerified: false,
+      link: "https://api.whatsapp.com/send?text="+window.location.href
```

Comment:
What's the reason for adding this to export block? Is it necessary to export this?

Reply:
resolved.